bindings/python/edlib.pyx

```diff
@@ -152,3 +154,98 @@ def align(query, target, mode="NW", task="distance", k=-1, additionalEqualities=
     cedlib.edlibFreeAlignResult(cresult)
 
     return result
+
+
+
+## function to output NICE alignments
+## only 'task=="path"' works
+
+def cigarToNiceAlignment(query, reference, mode="NW", k=-1, additionalEqualities=None):
```

Comment:
Hi @Martinsos 

The above looks reasonable. I guess I've been thinking about how I would use this function, though you are correct---developers should have access small, unit functions. 

I'll do the following:

```
query = "elephant"
target = "telephone"
result = align(query, target, task="path")
niceAlignment = cigarToNiceAlignment(result.cigar, query, target, gap = '-')
```

Reply:
Great :)!
Sorry if I am coming up as somewhat demanding / nitpicking, but we are under no deadline here so I think it is great opportunity to iterate together and try to make this code the best it can be :). And, I have been recently reading "Clean code" book so I am also trying to apply the advice from it.

Reply:
>  if I am coming up as somewhat demanding / nitpicking

Not at all! No problem whatsoever. :)

I think it makes sense

Reply:
Hi @Martinsos 
Thanks for the patience on this, and apologies it took forever for me to get back to you. 

I had a slight question:

```
query = "elephant"
target = "telephone"
result = align(query, target, task="path")
niceAlignment = cigarToNiceAlignment(result.cigar, query, target)  # Maybe some more inputs here? Ideally not.
```

It may be that I've forgotten some details, but it looks like we also need the `locations` field from `align()`. My recent commits therefore take the following arguments for `cigarToNiceAlignment()`:

```
def cigarToNiceAlignment(alignResult, query, target, gapSymbol="-"):
```
whereby `alignResult` is the entire output of `align()`. 

Am I missing something? It's quite possible :)